data/deerhold.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Deerhold",
+    "establishedAt": 1569246685,
+    "location": "Siphal, Kathmandu, Nepal",
+    "website": "https://deerhold.com/",
+    "email": "info@deerhold.org",
+    "phone": " 01-4593467",
+    "socials": {
+      "facebook": "https://www.facebook.com/deerhold",
+      "github": null,
+      "instagram": "https://www.instagram.com/deerhold/",
+      "linkedin": "https://www.linkedin.com/company/deerhold/",
+      "twitter": "https://twitter.com/DeerholdLtd",
+      "youtube": "https://www.youtube.com/@deerhold1841"
+    }
+  }
+  
```

Comment:
Please put a new line at the end of the file

Reply:
Resolved and added a new line at the end of the code